syntaxes/template.json

```diff
@@ -32,30 +32,60 @@
     },
 
     "propertyBinding": {
-      "begin": "(\\[.*\\])(=)(\"|')",
+      "begin": "(\\[\\s*@?[-_a-zA-Z0-9.$]*\\s*])(=)([\"'])",
       "beginCaptures": {
         "1": {
-          "name": "entity.other.attribute-name.html"
+          "name": "entity.other.attribute-name.html entity.other.ng-binding-name.html",
+          "patterns": [
+            {
+              "include": "#bindingKey"
+            }
+          ]
         },
         "2": {
           "name": "punctuation.separator.key-value.html"
         },
         "3": {
-          "name": "punctuation.definition.string.begin.html string.quoted.html"
+          "name": "string.quoted.html punctuation.definition.string.begin.html"
         }
       },
       "end": "\\3",
       "endCaptures": {
         "0": {
-          "name": "punctuation.definition.string.end.html string.quoted.html"
+          "name": "string.quoted.html punctuation.definition.string.end.html"
         }
       },
+      "name": "ng-binding.html",
```

Comment:
Same as above

Reply:
Updated! 0a254fe6fcd05a2f9fd60473328ed83ddd7d72ef

For this I took a cue from the HTML grammar and also added a `meta` tag to the scope (explanation [here](https://macromates.com/manual/en/language_grammars#naming_conventions)). The HTML grammar also inserts the property name into their equivalent scope with a backreference, which is something we can think about doing for the built-in directives like `ngStyle`/`ngClass` later on down the road.